<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>help</class>
 <widget class="QDialog" name="help">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>帮助界面</string>
  </property>
  <widget class="QTabWidget" name="Helpcontent">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>441</width>
     <height>181</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_help1">
    <attribute name="title">
     <string>帮助</string>
    </attribute>
    <widget class="QLabel" name="label_help">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>421</width>
       <height>171</height>
      </rect>
     </property>
     <property name="text">
      <string>1、没有收到取件密码短信怎么办？
可以拨打客服电话18200298768进行咨询处理。
2、不知道快递是否达到？
请在查询界面输入您的手机号码进行查询。
3、快递人员忘记密码怎么办？
请联系客服人员进行重置密码操作。
4、B1表示大号箱1号，M1表示中号箱1号，S1表示小号箱1号。
如有更多疑问，请致电客户服务热线18200298768！</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_help2">
    <attribute name="title">
     <string>关于</string>
    </attribute>
    <widget class="QLabel" name="label_about">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>421</width>
       <height>141</height>
      </rect>
     </property>
     <property name="text">
      <string>智能快递箱系统 v1.0
马凌
西南交通大学信息科学与技术学院计算机科学与技术专业 
20112399
253653617@qq.com</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_setting">
    <attribute name="title">
     <string>系统设置</string>
    </attribute>
    <widget class="QPushButton" name="pbSetting">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>110</y>
       <width>121</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>进入系统设置</string>
     </property>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>291</width>
       <height>131</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="pb1">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pb2">
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pb3">
        <property name="text">
         <string>3</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pb4">
        <property name="text">
         <string>4</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pb5">
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pb6">
        <property name="text">
         <string>6</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="pb7">
        <property name="text">
         <string>7</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="pb8">
        <property name="text">
         <string>8</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="pb9">
        <property name="text">
         <string>9</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="pbDelete">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>删除</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="pb0">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>10</y>
       <width>121</width>
       <height>55</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_setpassword">
        <property name="text">
         <string>系统设置密码</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_setpassword"/>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>441</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_help">
    <item>
     <spacer name="horizontalSpacer_help">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pbReturn_help">
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
